Simplenote/src/main/res/values-ar/strings.xml

```diff
@@ -65,11 +79,11 @@ Language: ar
     <string name="toast_email_sent">تم إرسال رسالة عبر البريد الإلكتروني</string>
     <string name="fullscreen_verify_email_button_secondary">إعادة إرسال بريد إلكتروني</string>
     <string name="fullscreen_verify_email_title">التحقُّق من بريدك الإلكتروني</string>
-    <string name="fullscreen_verify_email_subtitle">أرسلنا رسالة بريد إلكتروني خاصة بالتحقق إلى &lt;b>%1$s&lt;/b>. يرجى التحقُّق من علبة الوارد لديك واتباع الإرشادات.</string>
+    <string name="fullscreen_verify_email_subtitle">أرسلنا رسالة بريد إلكتروني خاصة بالتحقق إلى &lt;b&gt;%1$s&lt;/b&gt;. يرجى التحقُّق من علبة الوارد لديك واتباع الإرشادات.</string>
     <string name="fullscreen_review_account_button_secondary">تغيير البريد الإلكتروني</string>
     <string name="fullscreen_review_account_button_primary">تأكيد</string>
     <string name="fullscreen_review_account_title">مراجعة حسابك</string>
-    <string name="fullscreen_review_account_subtitle">لقد قمت بالتسجيل في Simplenote باستخدام البريد الإلكتروني &lt;b>%1$s&lt;/b>.&lt;br>&lt;br>قد تؤدي التحسينات التي يتم إجراؤها على أمان الحساب إلى فقدان الحساب إذا لم يعد لديك حق الوصول إلى عنوان البريد الإلكتروني هذا.</string>
+    <string name="fullscreen_review_account_subtitle">لقد قمت بالتسجيل في Simplenote باستخدام البريد الإلكتروني &lt;b&gt;%1$s&lt;/b&gt;.&lt;br&gt;&lt;br&gt;قد تؤدي التحسينات التي يتم إجراؤها على أمان الحساب إلى فقدان الحساب إذا لم يعد لديك حق الوصول إلى عنوان البريد الإلكتروني هذا.</string>
```

Comment:
Are these changes expected and okay? I wasn't sure if the > characters were supposed to be swapped out or not.

Reply:
Those are ok.

In fact, [the source XML for the English copy also uses `&gt;`](https://github.com/Automattic/simplenote-android/blob/95ac667b1f159a0d0c7271a128c87c3b89c7f12d/Simplenote/src/main/res/values/strings.xml#L322).